### DISCUSSION TITLE
Taliban Force 2021 (RHS & 3CB)

### DISCUSSION THREAD

**Discussion by OffDeadPixel** [Vdauphin/HeartsAndMinds#1172] [Question and answer]
Hello.

The current state of Taliban has been added to the Workshop under Taliban Force 2021 (RHS & 3CB)
I was Woundering if this can be added as a faction ?

**Comment by Vdauphin** [comment]
Hello,
May be 
link?
Cheers

**Reply by OffDeadPixel** [reply]
https://steamcommunity.com/sharedfiles/filedetails/?id=2585568796

**Comment by Vdauphin** [comment]
done here https://github.com/Vdauphin/HeartsAndMinds/pull/1231
